File: dice-game/dice.py
def DiceGame():
    import random

    play = True
    while play:
        number1 = random.randrange(1,7)
        number2 = random.randrange(1,7)
        print "You rolled a {} and a {}".format(number1,number2)
        print "Total: {}".format(number1+number2)
        cont_game = raw_input("Press 'Enter' to roll again, 'S' to stop ")
        print ""
        if cont_game == "S":
            play = False

DiceGame()

File: dice-game/dice.rb
# YOUR CODE, HERE
def DiceGame()
  while true
    num1 = rand(6)+1
    num2 = rand(6)+1

    puts "You rolled a #{num1} and a #{num2}"
    puts "Total: #{num1+num2}"
    puts "Press 'Enter' to roll again, 'S' to stop "
    cont_game = gets.chomp

    if cont_game == 'S'
      break
    end

  end
end

DiceGame()
